includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -62,4 +72,111 @@ public function get_summary( $request ) {
 		return rest_ensure_response( $summary );
 	}
 
+	/**
+	 * Renders HTML for a print receipt
+	 *
+	 * @return void
+	 */
+	public function generate_print_receipt() {
+		include_once WCPAY_ABSPATH . 'includes/in-person-payments/class-wc-payments-in-person-payments-receipt-service.php';
```

Comment:
What about my feedback (perhaps it was another PR) regarding proper shaped and injected service?